source/docs/api-reference/api-usage/actuator-limits.rst

```diff
@@ -40,8 +40,9 @@ The state of the forward or reverse limit switch can be retrieved from the API v
       .. code-block:: python
 
          import phoenix6
+         from phoenix6 import signals
 
-         forwardLimit = self.m_motor.get_forward_limit()
+         forward_limit = self.motor.get_forward_limit()
 
-         if (forwardLimit.value is phoenix6.CLOSED_TO_GROUND):
+         if (forward_limit.value is signals.CLOSED_TO_GROUND):
```

Comment:
Shouldn't you still need `signals.ForwardLimitValue.CLOSED_TO_GROUND`? I don't think enums are promoted to the global space, and I don't think we'd want to encourage using them like that anyways.